components/Theme/ThemeToggle/ThemeToggle.tsx

```diff
@@ -4,7 +4,7 @@ import { MoonIcon, SunIcon } from "@heroicons/react/20/solid";
 import { useTheme } from "next-themes";
 import { useEffect, useState } from "react";
 
-const THEME_MODES = {
+export const THEME_MODES = {
```

Comment:
Needed this exported so I could use it to determine the coco log that should be displayed. ie black or white